commit 8357d3662047df0005f1648c152c8433b56d3d51
Author: John Levon <john.levon@joyent.com>
Date:   2019-07-26T13:44:56+00:00 (2 months ago)
    
    TRITON-1838 headnode build should insist on answer-file
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/bin/build-usb-image b/bin/build-usb-image
index 4b53ec7c..fc31775c 100755
--- a/bin/build-usb-image
+++ b/bin/build-usb-image
@@ -113,11 +113,11 @@ function copy_config {
         # doc'd for a long time.
         ANSWER_FILE=$(build_spec answers-file)
     fi
-    if [[ -n ${ANSWER_FILE} && -f ${ANSWER_FILE} ]]; then
+    if [[ -n ${ANSWER_FILE} ]]; then
         # BASHSTYLED
         echo "WARNING: Copying in ${ANSWER_FILE}, remove answer-file from build.spec* to disable."
         mkdir -p ${MNT_DIR}/private
-        cp ${ANSWER_FILE} ${MNT_DIR}/private/answers.json
+        cp ${ANSWER_FILE} ${MNT_DIR}/private/answers.json || exit 1
     fi
 }
 
